src/forms/_base.scss

```diff
@@ -173,30 +120,26 @@ select {
 
 // the title element
 .form-legend {
-  padding-bottom: 10px;
-  margin-bottom: size(normal);
+  @include heading-2;
+  padding-bottom: ms(-2);
+  margin-bottom: ms(3);
+  color: color(text);
   border-bottom: 1px solid color(border);
   align-items: center;
 
-  @include respond-at-and-above($break-medium) {
-    padding-bottom: 15px;
-    margin-bottom: size(large);
-  }
-
-
-  @include respond-at-and-above($form-tablet) {
+  @include respond-at-and-above($form-max) {
     display: flex;
   }
 }
 
 .form-legend-alt {
-  @include body--sans-small;
-
+  font-size: ms(0);
+  letter-spacing: size(smaller-tracking);
```

Comment:
why do you need to state the letter spacing here?  it doesn't inherit an appropriate value from the body?